app/controllers/authorizer.rb

```diff
@@ -0,0 +1,56 @@
+module Authorizer
+  def tapp_admin
+    is_admin(ENV['TAPP_ADMINS'])
+  end
+
+  def cp_admin
+    is_admin(ENV['CP_ADMINS'])
+  end
+
+  def correct_applicant
```

Comment:
comment: the applicant authenticated by shib matches the utorid of the applicant the offer was made to.